crates/astria-sequencer/src/config.rs

```diff
@@ -14,6 +14,9 @@ pub struct Config {
     /// The path to the json encoded genesis file with a list of accounts.
     #[arg(long)]
     pub(crate) genesis_file: String,
+    /// The path to penumbra storage db.
+    #[arg(long)]
+    pub(crate) db_filepath: String,
```

Comment:
this is often called `datadir`, rename?

Reply:
I copied the name from https://github.com/penumbra-zone/penumbra/blob/43768e74c1e4a6bd8280dd3faeb487b791b49fc0/crates/storage/src/storage/temp.rs#L24

but looking at it deeper, I think that even though it's named `penumbra.db`, it is actually a folder. I'll update it!

Reply:
@steezeburger the original terminology was correct. `Storage::load` opens a binary file at the supplied path: https://rustdoc.penumbra.zone/main/src/penumbra_storage/storage.rs.html#46-130

Please rename back because calling this a directory is not correct.